shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/lock/ScalingSchemaNameDistributeLock.java

```diff
@@ -22,22 +22,21 @@
 import org.apache.shardingsphere.mode.manager.cluster.coordinator.lock.service.LockRegistryService;
 import org.apache.shardingsphere.mode.repository.cluster.ClusterPersistRepository;
 import org.apache.shardingsphere.mode.repository.cluster.ClusterPersistRepositoryConfiguration;
-import org.apache.shardingsphere.spi.typed.TypedSPIRegistry;
 
 import java.util.Map;
 
 /**
- * Zookeeper Distribute Lock.
+ * distributed locks added to the schema name during scaling.
  */
-public final class ZookeeperDistributeLock {
+public final class ScalingSchemaNameDistributeLock {
     
     private static final LockRegistryService LOCK_REGISTRY_SERVICE;
     
     private static final Map<String, Boolean> LOCK_NAME_LOCKED_MAP;
     
     static {
         ClusterPersistRepositoryConfiguration repositoryConfig = (ClusterPersistRepositoryConfiguration) PipelineContext.getModeConfig().getRepository();
-        ClusterPersistRepository repository = TypedSPIRegistry.getRegisteredService(ClusterPersistRepository.class, repositoryConfig.getType(), repositoryConfig.getProps());
+        ClusterPersistRepository repository = (ClusterPersistRepository) PipelineContext.getContextManager().getMetaDataContexts().getMetaDataPersistService().get().getRepository();
         repository.init(repositoryConfig);
```

Comment:
`repository` should be already initialized, is `repository.init(repositoryConfig)` necessary?